### ISSUE TITLE
App keeps running after the platform throws an error.

### ISSUE THREAD

**Issue by cerealpxl** [FosterFramework/Foster#65]
Hello
When the application creates a window with (0, 0) size, the console logs an error but the app keeps running and throwing more errors.

![image](https://github.com/FosterFramework/Foster/assets/83312057/ab49097b-ae21-4efe-a637-972e721473a3)

You can replicate it writing something like: 
~~~cs
App.Run("app", 0, 0);
~~~

**Comment by kuujoo** [comment]
0,0 is invalid value for size, causing the platform library to fail to start.

```
if (fstate.desc.width <= 0 || fstate.desc.height <= 0)
{
  FosterLogError("Foster invalid application width/height (%i, %i)", desc.width, desc.height);
  return;
}
```

Perhaps C# side could check if the platform library is running after the 'FosterStartup' call. If not throw C# exception or early exit.

**Comment by NoelFB** [comment]
Should be fixed now with #66, will throw an exception and stop.
